yml/dp02_dc2.yaml

```diff
@@ -8712,6 +8712,15 @@ tables:
     ivoa:ucd: meta.id;obs.field
     tap:principal: 0
     tap:column_index: 140
+  constraints:
+  - name: FSDO_DO
```

Comment:
Should be `DiaO_FSoDiaO` (the 1 side comes first, then the N side; the other names involving DiaXxxxx are shortened to `DiaX`)